src/sentry/api/endpoints/sentry_app_installation_external_issue_details.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework.response import Response
+
+from sentry.api.bases import (
+    SentryAppInstallationExternalIssueBaseEndpoint as ExternalIssueBaseEndpoint,
+)
+from sentry.mediators import external_issues
+from sentry.models import PlatformExternalIssue
+
+
+class SentryAppInstallationExternalIssueDetailsEndpoint(ExternalIssueBaseEndpoint):
+    def delete(self, request, installation, external_issue_id):
+        try:
+            platform_external_issue = PlatformExternalIssue.objects.get(
+                id=external_issue_id,
+                project__organization=installation.organization,
```

Comment:
@MeredithAnya You might consider using this query filter for the PR you added to avoid querying all projects for an org: https://github.com/getsentry/sentry/blob/a737df99746415489bea9a66c600a7e4f6488632/src/sentry/api/endpoints/sentry_app_installation_external_issues.py#L26